python/lsst/meas/algorithms/psfCandidate/__init__.py

```diff
@@ -0,0 +1,4 @@
+from __future__ import absolute_import
```

Comment:
But...really? They don't contain *any* interesting code. I'd personally skip it if they are not already present.